backend/src/schema/resolvers/users.js

```diff
@@ -177,6 +177,7 @@ export default {
         'termsAndConditionsAgreedVersion',
         'termsAndConditionsAgreedAt',
         'allowEmbedIframes',
+        'showShoutsPublicly',
```

Comment:
Very important that we not forget to add this! Great job catching this issue before it bites us @kachulio